vignettes/target_types.Rmd

```diff
@@ -0,0 +1,49 @@
+---
+title: "Target type guidelines"
+author: "David Watkins"
+date: "3/25/2019"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Target types}
+  %\VignetteEngine{knitr::rmarkdown}
+  \usepackage[utf8]{inputenc}
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+`scipiper` targets can be indicator files with a [shared cache](shared_cache.html), local data files with or without local indicator files, or simply R objects.  This vignette provides some guidelines for when to use these different patterns.
```

Comment:
The shared_cache vignette covers it in more detail, but I think it'd be useful in this vignette to remind readers of how indicator file + data file pairs get used and why they work. The most important point, I think, is that when using indicator/data pairs, the workflow dependencies are connected via indicator files and not by data files, which means that remake ensures the local up-to-dateness of the indicator files, but corresponding data files don't necessarily need to be local and up to date, saving team members from unnecessary downloads or rebuilds of data files they're not using directly.

Reply:
(That idea could surely be rephrased into shorter, more readable sentences than what I just wrote.)